### PR TITLE
git-imerge 1.2.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/git-imerge.rb [hunk 1]
```diff
@@ -16,13 +18,57 @@ class GitImerge < Formula
...
+    expected_output = <<~EOS
+      Attempting automerge of 1-1...success.
+      Autofilling 1-1...success.
+      Recording autofilled block MergeState('master', tip1='test', tip2='master', goal='merge')[0:2,0:2].
+      Merge is complete!
+    EOS
+    assert_match expected_output, shell_output("#{bin}/git-imerge merge master 2>&1")
   end
 end
+
```
Comment (SMillerDev) [inline Formula/git-imerge.rb:58]: Is this submitted upstream?

Comment (chenrui333) [reply Formula/git-imerge.rb:58]: yes, indeed. PR ref, https://github.com/mhagger/git-imerge/pull/176

Comment (chenrui333) [reply Formula/git-imerge.rb:58]: And it got merged as well, I will add as comments.

Comment (SMillerDev) [reply Formula/git-imerge.rb:60]: Can we use the upstream patch then

Comment (chenrui333) [reply Formula/git-imerge.rb:60]: I cannot directly apply the patch (I tried that)

Comment (chenrui333) [reply Formula/git-imerge.rb:60]: @SMillerDev I think we can shoot for the merge for this PR?

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**